src/main/java/org/cactoos/text/TextOf.java

```diff
@@ -317,17 +318,14 @@ public TextOf(final String input) {
      * Ctor.
      *
      * @param iterable The iterable to convert to string
-     * @todo #1461:30min We want constructors with {@code Iterable<?>} and
-     *  {@code Iterator<Character>} to have same behaviour (simply concatenate list of strings).
-     *  To do that, we should change {@code Iterable<?>} to {@code Iterable<Character>}
-     *  and delegate the {@link Iterator} one to the {@link Iterable} one. After that, we need to
-     *  modify other parts of cactoos that relied on this in order to preserve their behaviour
-     *  by using {@link Joined} and {@link Concatenated}.
-     */
-    public TextOf(final Iterable<?> iterable) {
+     * @todo #1560:30min/DEV We want {@link Concatenated} to have
+     *  an extra constructor that accepts {@code Iterable<CharSequence>}
+     *  to avoid creating of {@link Joined} with empty delimiter.
```

Comment:
@DmitryBarskov actually it won't be possible to introduce such constructor because of type erasure, so I propose to simply write the following in the constructor below instead: `new Concatenated(new Mapped(TextOf::new, iterable))`